source/Calamari/Log.cs

```diff
@@ -32,6 +33,18 @@ public static void Verbose(string message)
             }
         }
 
+        public static void SetVariable(string name, string value)
```

Comment:
Perhaps this method should be named SetOutputVariable or something similar, given it doesn't actually set the variable.  Or perhaps an extension method the VariableDictionary that both sets the variable and writes the log message?